moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -323,6 +344,7 @@ class Robot
   bool getFK(const std::vector<double> & joint_point,
                                Eigen::Affine3d &pose) const;
 
+  /*Lookup Trajectory Point in the Robot Model*/
```

Comment:
Document this method or remove the comment.

Reply:
Ok